HTML/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="testingApp">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Testing Project</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body ng-controller="MainController as mainCtrl">
+
+    <div class="level1">
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/17#issuecomment-2189244114>#automa</a><span> - 06/25/2024, 03:34 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Deeply nested divs (up to 5 levels) can lead to rendering performance issues and are not scalable for complex UIs.
</div>
<div id="fix">
<b>Fix:</b> Simplify the HTML structure to reduce the nesting levels.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -11,11 +11,7 @@
      <div class="level2">
          <div class="level3">
              <div class="level4">
                  <div class="level5">
                      Nested Divs up to 5 levels
                  </div>
              </div>
          </div>
      </div>
```
</div>
</details>
</div>
<br>


</div>